contracts/strategies/pangolin/strategy-png-avax-ust.sol

```diff
@@ -80,18 +80,11 @@ contract StrategyPngAvaxUst is StrategyPngMiniChefFarmBase {
         }
 
         // 10% is sent to treasury
-        uint256 _png = IERC20(png).balanceOf(address(this));
         uint256 _ust = IERC20(ust).balanceOf(address(this));
+        uint256 _wavax = IERC20(wavax).balanceOf(address(this));
+        uint256 _png = IERC20(png).balanceOf(address(this));
         uint256 _luna = IERC20(luna).balanceOf(address(this));
-        if (_png > 0) {
-            uint256 _keep = _png.mul(keep).div(keepMax);
-            if (_keep > 0) {
-                _takeFeePngToSnob(_keep);
-            }
-
-            _png = IERC20(png).balanceOf(address(this));  
-        }
-
+        
         if (_ust > 0) {
             uint256 _keep2 = _ust.mul(keep).div(keepMax);
```

Comment:
It looks like we are setting the 'keep' value right before we use it each time (_takeFeeXYZToSnob()) so there is no need to specifically call this 'keep' variable 'keep2'